src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -12,36 +12,37 @@
 import java.util.List;
 
 @RestController
+@RequestMapping("/lines")
 public class LineController {
 	private final LineService lineService;
 
 	public LineController(LineService lineService) {
 		this.lineService = lineService;
 	}
 
-	@PostMapping("/lines")
+	@PostMapping("")
```

Comment:
컨트롤러에 RequestMapping을 선언한 value를 그대로 사용할 경우 별도의 value를 지정해주지 않으셔도 됩니다 😃 
```suggestion
	@PostMapping
```